static/app/components/teamSelector.tsx

```diff
@@ -154,6 +178,45 @@ function TeamSelector(props: Props) {
     }
   }
 
+  const createTeam = () =>
```

Comment:
shall we use `useCallback` here so this is not recreated in every render?

Reply:
Not sure. personally, I am not a fan of using `useCallback` before there is a problem... For it to really have an effect, we would need to wrap like also all the other functions and the generation of the options array with `useMemo` and `useCallback`.

Reply:
I read Kent C. Dodds blog post on this topic too :) but folks from the tsc were pushing to use `useCallback` in these cases. Maybe that is something you can bring up in the next meeting 😉

Reply:
cc @JonasBa

Reply:
k, but then I would do it in a separate ref() PR, if that is ok.
The current changes should not worsen the re-rendering behaviour as it already creates a new options array on every render.

Reply:
fair enough 😉